test/test_other.py

```diff
@@ -12435,10 +12341,8 @@ def test(args):
     print('sizes', normal, changed)
     # Changing this option to [] should decrease code size.
     self.assertLess(changed, normal)
-    # Check an absolute code size as well, with some slack.
-    self.check_expected_size_in_file('js',
-                                     test_file('other/test_INCOMING_MODULE_JS_API.js.size'),
-                                     changed)
+    # Check an absolute code size as well.
+    self.checkOutputSizes(['a.out.js'])
```

Comment:
We use `snake_case` for all our internal helper methods (even though the `assertFoo` family of functions uses camel.

Reply:
From AI:

```
Adhering to PEP 8 in unittest:

While unittest itself uses camelCase for its built-in methods (like assertEqual and setUp), which predates the strong emphasis on snake_case in PEP 8 for functions and methods, your custom test methods and any helper methods within your test classes should still adhere to PEP 8's snake_case convention for readability and consistency within your own
```